### PR TITLE
Register remaining benchmarks

### PR DESCRIPTION
**Pull request by sea212**
Preparation for #377 

Before we are able to benchmark every pallet (including external pallets), we have to register their benchmarking logic in our runtime.
The frame pallets work out-of-the-box, the orml pallet unfortunately require a custom benchmarking environment and benchmarking logic.

### CODE REVIEW

**Comment by sea212** [comment]
Blocked until #377 is merged.

**Comment by sea212** [comment]
Rebasing onto main soon.

**Review by lsaether — APPROVED** [review]

**Pull request merged**
